tox.ini

```diff
@@ -0,0 +1,19 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
```

Comment:
Just a question. We do not have github actions in lsp-elm repo. Is this file intended to be run manually in the terminal for now or?

Reply:
I'm running flake8 locally through LSP so I saw issues with too long lines without it.